Source/ocdm/CMakeLists.txt

```diff
@@ -34,12 +34,9 @@ find_package(${NAMESPACE}Core REQUIRED)
 find_package(${NAMESPACE}COM REQUIRED)
 find_package(${NAMESPACE}Messaging REQUIRED)
 
-find_package(GSTREAMER REQUIRED)
-find_package(GSTREAMER_BASE REQUIRED)
+set(CDMI_ADAPTER_IMPLEMENTATION "None" CACHE STRING "Defines which implementation is used.")
 
-option(CDMI_ADAPTER_IMPLEMENTATION "Defines which implementation is used." "None")
-
-add_library(${TARGET} 
+add_library(${TARGET} SHARED
```

Comment:
Please don't force to build a shared library, let the build system configuration decide what type of library is build